src/components/Table/NodeDetails.tsx

```diff
@@ -83,7 +83,7 @@ const NodeDetails: FC<NodeDetailsProps> = ({ nodeData, onClose }) => {
             </Typography>
             </Grid>
             <Grid item xs={12}>
-              <Typography variant="subtitle1"><strong>Reward Eligibility:</strong> {nodeData?.eligible?.toLocaleString()}</Typography>
+              <Typography variant="subtitle1"><strong>Last Round Check Eligibility:</strong> {nodeData?.eligible?.toLocaleString()}</Typography>
```

Comment:
```suggestion
              <Typography variant="subtitle1"><strong>Last Round Eligibility Check:</strong> {nodeData?.eligible?.toLocaleString()}</Typography>
```

Reply:
To be honest, IMO, i'm not sure is this new wording is helpful. I still found it a bit confusing.
As long as we have a red cross there, people will just interpret it like "It is Not Eligible"
I think this will be more clear once we have a green check mark.. or we could have a warning sign instead, if the last check failed, but the node is still eligible given its uptime ... Maybe extra details/explanation could be on some hover tooltip

Reply:
Yeah, I agree. It is (at most) just a small amount better than before. It's unlikely to stop the complaints on Discord... We can update the tooltip but I agree with Mihai that it's unlikely anyone will read it.

I think this is better than nothing as something we can quickly merge, and then we should work on displaying uptime based on if the node has 90% uptime, as I said in my other comment.

Reply:
i have updated the tooltip message as well, check this change in the table index
`<Tooltip title="These nodes were eligible to receive rewards the proportion of their uptime 
           at the last round checks.">
          <span>Last Check Eligibility</span>
        </Tooltip>`

Reply:
I have another suggestion on this. Rather than displaying the last check, we could show the last 5 checks. That at least puts a failed check in context, so it is slightly less worrying seeing one out of 5 checks fail vs a single red cross for the last (and only visible) check. 

For example: 

![image](https://github.com/user-attachments/assets/cc7e9b95-a340-4b21-9053-4e256217fc0f)

Reply:
we can't really do that kind of updates to the dashboard without some heavy lifting to the backend, we have only last check stored for each node, will imply some changes to the things we store in es and how we are retrieving the nodes

Reply:
Ok, it is possibly something for later then. I'll suggest it in a separate issue.

Reply:
yep we can have it with monitoring v2 maybe, cause that will also require changes over there